internal/component/database_observability/mysql/collector/query_tables.go

```diff
@@ -111,19 +111,19 @@ func (c *QuerySample) Start(ctx context.Context) error {
 	return nil
 }
 
-func (c *QuerySample) Stopped() bool {
+func (c *QueryTables) Stopped() bool {
 	return !c.running.Load()
 }
 
 // Stop should be kept idempotent
-func (c *QuerySample) Stop() {
+func (c *QueryTables) Stop() {
 	c.cancel()
 }
 
-func (c *QuerySample) fetchQuerySamples(ctx context.Context) error {
-	rs, err := c.dbConnection.QueryContext(ctx, selectQuerySamples)
+func (c *QueryTables) fetchQueryTables(ctx context.Context) error {
+	rs, err := c.dbConnection.QueryContext(ctx, selectQueryTablesSamples)
 	if err != nil {
-		level.Error(c.logger).Log("msg", "failed to fetch query samples", "err", err)
+		level.Error(c.logger).Log("msg", "failed to fetch query tables", "err", err)
```

Comment:
Maybe this log line can stay as is? Because it's actually still fetching samples here.

Reply:
The reason I renamed this was that we need to avoid the collision when we introduce the new query sample collector.

The purpose of this query is to fetch the table info - the fact it is still fetching sample info is a caveat of not yet removing that logic here.

Reply:
Ok I see the point on log collision, but this is actually fetching samples still, not tables. Maybe change it to something line `failed to fetch query samples from events_statements_summary_by_digest`?

Reply:
Updated in 8d3f533f82cb44c7263add010a6c700bbfaa70ca